src/models/generated/HydroDispatch.jl

```diff
@@ -10,6 +10,8 @@ This file is auto-generated. Do not edit.
         reactivepower::Float64
         tech::TechHydro
         op_cost::TwoPartCost
+        storagecapacity::Float64
```

Comment:
we might need to add an inflow field as well here.

Reply:
ok, I can do that. Should `storage_capacity` and `inflow`fields be in the TechHydro ?